14/umbraco-cms/tutorials/creating-a-basic-website/getting-started.md

```diff
@@ -1,13 +1,13 @@
 # Getting Started
 
-The **Creating a Basic Site** tutorial provides step-by-step instructions to build an Umbraco website using a set of HTML, CSS, and JavaScript files. The tutorial allows you to use a website template, customize it, and wire up the sections that need content managing in the Umbraco CMS.
+The **Creating a Basic Site** tutorial provides step-by-step instructions to build an Umbraco website using a set of HTML, CSS, and JavaScript files. This tutorial enables you to use a website template, modify it, and connect the sections that require content management in the Umbraco CMS.
 
 ## **What You Need**
 
-To install a basic website in the Umbraco CMS, you need the following:
+To begin this tutorial, you will need to:
```

Comment:
How about something like "You have two options"?

Also, should we even suggest that they can use another HTML template?
Wouldn't it be difficult to follow the tutorial without the files it was built around? 🤔

Reply:
That's a good point. I'll take out the other option to avoid any difficulties 😀